src/Integration/Laravel/Cache/src/AsyncAwsDynamoDbStore.php

```diff
@@ -377,7 +379,35 @@ public function forget($key)
      */
     public function flush()
     {
-        throw new RuntimeException('DynamoDb does not support flushing an entire table. Please create a new table.');
+        try {
+            // Delete the table
+            $this->dynamoDb->deleteTable(['TableName' => $this->table]);
+        } catch (HttpException $e) {
+            if ($e->getCode() !== 404) {
+                // Any error but "table not found"
+                throw new \RuntimeException('Could not flush DynamoDb cache. Table could not be deleted.', $e->getCode(), $e);
+            }
+        }
+
+        // Wait until table is removed
+        if (!$this->dynamoDb->tableNotExists(['TableName'=>$this->table])->isSuccess()) {
```

Comment:
should call `wait` before. otherwise, you'll get `false`of the table is not yet deleted

Reply:
Thank you